metadata_backend/api/handlers/object.py

```diff
@@ -263,6 +285,10 @@ async def put_object(self, req: Request) -> Response:
         patch = await self.prepare_folder_patch_update_object(collection, accession_id, title, filename)
         await folder_op.update_folder(folder_id, patch)
 
+        # Update draft dataset to Metax catalog
+        if collection in {"study", "dataset"}:
+            await self.create_or_update_metax_dataset(req, collection, accession_id)
```

Comment:
I understand the need to keep this up to date, however if we update only once when we publish would that be bad ?
if we had only 2 calls to Metax:
1. to create draft Metax URL
2. to finalize/update information when we publish the folder